src/backend/src/routes/design-reviews.routes.ts

```diff
@@ -0,0 +1,31 @@
+import express from 'express';
+import { body } from 'express-validator';
+import { intMinZero, nonEmptyString, isDate, isDesignReviewStatus } from '../utils/validation.utils';
+import DesignReviewController from '../controllers/design-reviews.controllers';
+
+const designReviewRouter = express.Router();
+
+designReviewRouter.post(
+  '/:designReviewId/edit',
+  isDate(body('dateScheduled')),
+  nonEmptyString(body('teamType')),
+  body('requiredMembers').isArray(),
+  intMinZero(body('requiredMembers.*')),
+  body('optionaldMembers').isArray(),
+  body('isOnline').isBoolean(),
+  body('isInPeron').isBoolean(),
+  nonEmptyString(body('zoomLink')).isURL().optional,
```

Comment:
make sure you include the parentheses on .optional so you invoke the handler i.e. it should be nonEmptyString(body('zoomLink')).isURL().optional()

Reply:
the same goes with the other instances of `.optional`